server/enhancedDate.js

```diff
@@ -1 +1,126 @@
-// export this as a Node module using the enhancedDate.js code from Module_2
\ No newline at end of file
+var enhancedDate = function () {    
```

Comment:
in your Node module version of this file, you're not setting enhancedDate to a function expression, you're returning an object with all of the enhancedDate methods on it. So at the bottom, set module.exports equal to an object with all of those methods on it: var enhancedDate = { getDate: getDate, setDate: setDate, etc. }; module.exports = enhancedDate;`